test/lib/utils/addLiquidityBoostedHelper.ts

```diff
@@ -130,3 +136,170 @@ export const assertTokenMatch = (
         expect(a.decimals).to.eq(tokenReturned[i].decimals);
     });
 };
+
+export type AddLiquidityBoostedTxInput = {
+    client: PublicWalletClient & TestActions;
```

Comment:
Could including `TestActions` type here cause problem for SDK consumer's who want to send tx's to live networks?

Reply:
This type is used only within tests, so it should be ok.

Reply:
Err nvm i forgot SDK abstracts away complexity of boosted and only asks user for `AddLiquidityInput` and `AddLiquidityKind`